### DISCUSSION TITLE
get_author_feed through different times

### DISCUSSION THREAD

**Discussion by teddy-masters** [MarshalX/atproto#462] [Questions] [answered]
I understand that we can't get more than 100 entries from get_author_feed, that is okay. Is there a way to get the posts from an author that aren't in the most recent 100? For example the oldest 100, or 100 in the middle. Thanks!

**Comment by MarshalX** [comment, marked as answer]
Why we can't get more than 100 entries? 100 is default "per page". Such endpoints should use pagination mechanism. See the example how to deal with it: https://github.com/MarshalX/atproto/blob/main/examples/advanced_usage/handle_cursor_pagination.py
